src/components/MapSelectors/datasets/us_states_dataset_01_02_2020.json

```diff
@@ -48022,6 +48028,62 @@
         }
       ]
     },
+    "MP": {
+      "dataset": [
+        {
+          "counties": [
+            "Northern Islands Municipality",
+            "Rota Municipality",
+            "Saipan Municipality",
+            "Tinian Municipality"
+          ],
+          "population": 107766
+        }
+      ],
+      "county_dataset": [
+        {
+          "county": "Northern Islands Municipality",
+          "county_url_name": "northern_islands_municipality",
+          "county_fips_code": "085",
+          "state_fips_code": "69",
+          "state_code": "MP",
+          "full_fips_code": "69085",
+          "cities": [""],
```

Comment:
nit: Do we need a at least one entry, or could this just be `cities: []` ?

Reply:
That's a good question - I was mostly just copying what we had from the other entries